docs/homepage-source/docs/modules/ROOT/pages/index.adoc

```diff
@@ -13,20 +13,20 @@ link:https://www.youtube.com/watch?v=-9pVwCkkE1I[Watch on YouTube ,role=yt-widge
 With its powerful abstractions, Cloudflow allows you to easily define the most complex streaming applications:
 
 * Develop: Focus only on business logic, leave the boilerplate to us.
-* Build: We provide all the tooling for going from business logic to a deployable Docker image.
-* Deploy: We provide Kubernetes tooling to deploy your distributed system with a single command, and manage durable connections between processing stages.
-* Operate: With a Lightbend subscription, you get all the tools you need to provide insights, observability, and lifecycle management for evolving your distributed streaming application.
+* Build: We provide a rich development kit for going from business logic to a deployable application.
+* Deploy: Cloudflow comes with Kubernetes extensions for the cluster and the client to deploy your distributed system with a single command.
 
+Cloudflow integrates with popular streaming engines like Akka, Spark and Flink. 
+It also comes with a powerful CLI tool to easily manage, scale and configure streaming applications at runtime. 
+In a nutshell, Cloudflow is an application development toolkit composed of:
 
-Cloudflow integrates with popular streaming engines like Akka, Spark and Flink. It also comes with a powerful CLI tool to easily manage, scale and configure streaming applications at runtime. In a nutshell, Cloudflow is an application development toolkit composed of:
-
-* An API definition for `Streamlet`, the core abstraction in Cloudflow.
-* An extensible set of runtime implementations for `Streamlet`(s). Cloudflow provides support for popular streaming runtimes, like Spark's Structured Streaming, Flink, and Akka.
+* A definition for `Streamlet`, the core component model in Cloudflow.
+* An extensible runtime for `Streamlet`(s) and out-of-the-box implementations for popular streaming runtimes, like Spark's Structured Streaming, Flink, and Akka.
```

Comment:
```suggestion
* An extensible runtime for `Streamlet`(s) and out-of-the-box implementations for popular streaming runtimes, like Spark Structured Streaming, Flink, and Akka Streams.
```

Reply:
I think we had Akka Streams before and Craig asked to make it just "Akka". 
WDYT @RayRoestenburg @longmuir ?